Packs/CortexDataLake/README.md

```diff
@@ -0,0 +1,836 @@
+## Overview
+---
+
+Palo Alto Networks Cortex Data Lake provides cloud-based, centralized log storage and aggregation for your on premise, virtual (private cloud and public cloud) firewalls, for Prisma Access, and for cloud-delivered services such as Cortex XDR
+This integration was integrated and tested with version 2 of Cortex Data Lake
```

Comment:
is that correct version?

Reply:
Yes, as inferred from here 
![image](https://user-images.githubusercontent.com/41257953/77740998-41e74d80-7025-11ea-8e40-30269290501f.png)